core/templates/services/local-storage.service.spec.ts

```diff
@@ -121,5 +125,76 @@ describe('LocalStorageService', () => {
       expect(localStorageService.getLastSelectedTranslationTopicName())
         .toBeNull();
     });
+
+    it('should not get entity editor browser tabs info from local ' +
+    'storage when storage is not available', () => {
+      spyOn(localStorageService, 'isStorageAvailable').and.returnValue(false);
+      expect(localStorageService.getEntityEditorBrowserTabsInfo(
+        EntityEditorBrowserTabsInfoDomainConstants
+          .OPENED_TOPIC_EDITOR_BROWSER_TABS, 'topic_1')).toBeNull();
+    });
+
+    it('should add entity editor browser tabs info', () => {
+      const entityEditorBrowserTabsInfoObject:
+        EntityEditorBrowserTabsInfoObject = {
+          entityType: 'topic',
+          id: 'topic_1',
+          latestVersion: 1,
+          numberOfOpenedTabs: 1,
+          someTabHasUnsavedChanges: false
+        };
+      localStorageService.updateEntityEditorBrowserTabsInfo(
+        EntityEditorBrowserTabsInfo.createFromDict(
+          entityEditorBrowserTabsInfoObject),
+        EntityEditorBrowserTabsInfoDomainConstants
+          .OPENED_TOPIC_EDITOR_BROWSER_TABS,
```

Comment:
Don't think you need the comma at the end here. Ditto elsewhere.

Reply:
Removed.